docs-src/cli/cmd-options/metrics-port.md

```diff
@@ -10,7 +10,7 @@ tags:
     - command-line-interface-cli
     - logging and metrics
     - metrics
-
 ---
 
-Port for /metrics (default: 0).
+Port for `/metrics`.
+Disabled by default.
```

Comment:
After some test I've verified the metrics endpoint is enabled by default but with a random port that's never advertised. I've created a issue here for the cli repo to track.

https://github.com/temporalio/cli/issues/309

Reply:
Okay, I'll change this then.